db/migrations/2020_04_07_000000_change_mysql_database_encoding_to_utf8mb4.php

```diff
@@ -27,12 +27,4 @@ public function up(): void
             );
         }
     }
-
-    /**
-     * Reverse the migration
-     */
-    public function down(): void
-    {
-        // As utf8mb4 is a superset of utf8, there is nothing to do here
```

Comment:
Imho that comment is relevant and should stay even after removing the method.

Reply:
changed